ballerina/http_interceptors.bal

```diff
@@ -36,6 +36,16 @@ public type ResponseErrorInterceptor distinct service object {
 
 };
 
+# The service type to be used when engaging interceptors at the service level
+public type InterceptableService distinct service object {
+    *Service;
+
+    # Function to create and return the interceptors
+    #
+    # + return - The `http:Interceptor[]`
+    public function createInterceptors() returns Interceptor[];
```

Comment:
```suggestion
    public function createInterceptors() returns Interceptor|Interceptor[];
```
With the [recent changes](https://github.com/ballerina-platform/ballerina-standard-library/issues/3969) we have to allow `Interceptor` or `Interceptor[]` here

Reply:
Updated